scheduleit/views/meetings/remove_attendee.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+require_once ABSPATH . 'config/session.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
+    
+     $eventId = $_POST['eventId'];
+     $removeOnid = $_POST['attendeeOnid'];
+     $startTime = $_POST['startTime'];
+     $eventName = $_POST['meetingName'];
+
+     $slotId = $database->getSlotId($startTime, $eventId);
+
+     $result = $database->deleteBooking($startTime, $eventId, $removeOnid);
+
+     if ($result > 0) {
+         $user = $database->getUserByONID($removeOnid);
+         //create message
+         $msgFormat =
```

Comment:
If you look in my PR #44, I introduce a `SendEmail` class where I think all emails we send should be gathered. Plus, I formatted the emails to have a reply-to email.

Reply:
i'm removing my email code and puting a note to set up yours.